sourmash/commands.py

```diff
@@ -696,10 +696,17 @@ def gather(args):
             notify('no unassigned hashes! not saving.')
         else:
             notify('saving unassigned hashes to "{}"', args.output_unassigned)
-
+            moltype_kwds = {}
+            if args.protein or args.hp or args.dayhoff:
+                moltype_kwds['is_protein'] = True
+                if args.hp:
+                    moltype_kwds['hp'] = True
+                elif args.dayhoff:
+                    moltype_kwds['dayhoff'] = True
+                  
             with_abundance = next_query.minhash.track_abundance
             e = MinHash(ksize=query.minhash.ksize, n=0, max_hash=new_max_hash,
-                        track_abundance=with_abundance)
+                        track_abundance=with_abundance, **moltype_kwds)
```

Comment:
hmm, I'm wondering if it might be possible to create one with a "e = query.minhash.copy_and_clear()" instead, which will clone the attributes of the query MinHash? The only thing I can see being problematic is that you probably want to use `new_max_hash` instead of the `query.minhash.max_hash`, in case the signature's been downsampled. Hmm.

Reply:
it turns out this wasn't even necessary - `next_query` is actually the right thing to save directly! w00t!
see  https://github.com/dib-lab/sourmash/pull/1156